omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/RestTestConstants1_9.java

```diff
@@ -51,8 +51,6 @@
 	
 	public static final String CONCEPT_UUID = "c607c80f-1ea9-4da3-bb88-6276ce8868dd";
 	
-	public final static String ENCOUNTER_ROLE_UUID = "a0b03050-c99b-11e0-9572-0800200c9a66";
```

Comment:
Why did you remove this?

Reply:
It was being duplicated, I already have it in the `RestTestConstans1_11.java` file